test_scripts/API/SetAudioStreamingIndicator/ideas_SetAudioStreamingIndicator_conditions_for_SDL_to_handle_related_cases.txt

```diff
@@ -0,0 +1,111 @@
+./test_scripts/API/SetAudioStreamingIndicator/074_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_FULL_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_navi_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/075_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_LIMITED_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_navi_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/076_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_BACKGROUND_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_navi_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/077_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_NONE_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_media_navi_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/078_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_comm_app_FULL_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_comm_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/079_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_comm_app_LIMITED_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_comm_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/080_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_comm_app_BACKGROUND_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_comm_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/081_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_comm_app_NONE_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_media_comm_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/082_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_media_app_null_permission_ExtProprietary.lua Check_that_SDL_responds_with_DISALLOWED_when_media_app_has_null_permissions
+./test_scripts/API/SetAudioStreamingIndicator/083_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_null_permission_ExtProprietary.lua Check_that_SDL_responds_with_DISALLOWED_when_non-media_app_has_null_permissions
+./test_scripts/API/SetAudioStreamingIndicator/084_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_null_permission_ExtProprietary.lua Check_that_SDL_responds_with_DISALLOWED_when_navi_app_has_null_permissions
+./test_scripts/API/SetAudioStreamingIndicator/085_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_comm_app_null_permission_ExtProprietary.lua Check_that_SDL_responds_with_DISALLOWED_when_comm_app_has_null_permissions
+./test_scripts/API/SetAudioStreamingIndicator/086_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_media_app_null_permission_Proprietary.lua Check_that_SDL_responds_with_DISALLOWED_when_media_app_has_null_permissions
```

Comment:
@dboltovskyi, to be application revoked, you should perform successful PTU, To be done this, the full flow should be executed. Flows for Proprietary and HTTP are different. 
The requirement is the same, but preconditions for verification of it are different.